packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -440,16 +441,28 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
         // To set that up we start each client in a staggered way, each will independently go thru write
         // and listen cycles
 
-        const cycleMs = config.testConfig.readWriteCycleMs;
-        let t: NodeJS.Timeout | undefined;
+        let timeout: NodeJS.Timeout | undefined;
```

Comment:
in the future try to avoid mixing refactoring with changes, it makes it harder to review as i need to parse refactors from changes. if you really want  a refactor just push a small pr with only that. generally quick and easy to review as long as it is non-controversial like this

Reply:
Sounds good, I'll keep that in mind :")